include/zephyr/sys/util.h

```diff
@@ -387,6 +387,22 @@ extern "C" {
 #define MAX(a, b) (((a) > (b)) ? (a) : (b))
 #endif
 
+/** @brief Return larger value of two provided expressions.
```

Comment:
Typo in the commit message of 150cd2f1eb3344df0a3ab6b697e22b231877e54d
"ypeof" --> "typeof"